src/queries/circle/isMember.ts

```diff
@@ -0,0 +1,24 @@
+import { CIRCLE_ACTION } from 'common/enums'
+import { CircleToIsMemberResolver } from 'definitions'
+
+const resolver: CircleToIsMemberResolver = async (
+  { id },
+  _,
+  { viewer, dataSources: { atomService } }
+) => {
+  if (!viewer.id) {
+    return false
+  }
+
+  const record = await atomService.findFirst({
```

Comment:
same as the comment above, subscription item as the source of truth?

Reply:
Fixed.